src/app/_components/most-popular-section/tabs.tsx

```
@@ -0,0 +1,69 @@
+  return (
+    <Tabs defaultValue={topFour[0]._id} className="w-full">
+      <TabsList className="flex justify-end gap-4 mb-6 bg-transparent mt-[-30px] mb-14">
+        {/* render a tab button for each top occasion so the user can switch between different occasion tabs */}
+        {topFour.map((occasion) => (
+          <TabsTrigger key={occasion._id} value={occasion._id} className="text-base text-zinc-700 dark:text-zinc-400 font-medium  data-[state=active]:text-maroon-600 dark:data-[state=active]:text-softpink-200 data-[state=active]:shadow-none p-1.5">
+            {occasion.name}
+          </TabsTrigger>
+        ))}
+      </TabsList>
+    {/* loop through the top occasions and render a tab for each one with its products */}
+      {productsByOccasion.map(({ _id, name, products }) => (
```

ESLint

[error] 'name' is defined but never used.

(@typescript-eslint/no-unused-vars)